### ISSUE TITLE
loss.backward() RuntimeError

### ISSUE THREAD

**Issue by Asthestarsfalll** [MichaelFan01/STDC-Seg#30]
Hi,
I want to train the STDC with the other datasets,howerver it can only run for a short time
![image](https://user-images.githubusercontent.com/72954905/127307038-ef3c9e30-e9e1-4eef-98ae-0405bdb38c07.png)
I have assgined the `drop_last=True`
![image](https://user-images.githubusercontent.com/72954905/127309538-f73d0493-a4be-4a70-987d-4472b70ed520.png)

**Comment by MichaelFan01** [comment]
> Hi,
> I want to train the STDC with the other datasets,howerver it can only run for a short time
> ![image](https://user-images.githubusercontent.com/72954905/127307038-ef3c9e30-e9e1-4eef-98ae-0405bdb38c07.png)
> I have assgined the `drop_last=True`
> ![image](https://user-images.githubusercontent.com/72954905/127309538-f73d0493-a4be-4a70-987d-4472b70ed520.png)

有脏数据吧，你把图像大小都resize成32可除的固定大小呢~

**Comment by Asthestarsfalll** [comment]
> > Hi,
> > I want to train the STDC with the other datasets,howerver it can only run for a short time
> > ![image](https://user-images.githubusercontent.com/72954905/127307038-ef3c9e30-e9e1-4eef-98ae-0405bdb38c07.png)
> > I have assgined the `drop_last=True`
> > ![image](https://user-images.githubusercontent.com/72954905/127309538-f73d0493-a4be-4a70-987d-4472b70ed520.png)
> 
> 有脏数据吧，你把图像大小都resize成32可除的固定大小呢~
都已经固定为[480,832]的大小了，仍然是这个错误，555~

**Comment by Asthestarsfalll** [comment]
> > Hi,
> > I want to train the STDC with the other datasets,howerver it can only run for a short time
> > ![image](https://user-images.githubusercontent.com/72954905/127307038-ef3c9e30-e9e1-4eef-98ae-0405bdb38c07.png)
> > I have assgined the `drop_last=True`
> > ![image](https://user-images.githubusercontent.com/72954905/127309538-f73d0493-a4be-4a70-987d-4472b70ed520.png)
> 
> 有脏数据吧，你把图像大小都resize成32可除的固定大小呢~

都已经固定为[480,832]的大小了，仍然是这个错误，555~

**Comment by gaowq2017** [comment]
如果图像固定了，可以尝试把crop增强屏蔽掉，以及可以改变尺寸的都屏蔽掉，亲试有效

**Comment by Asthestarsfalll** [comment]
不知道为什么是ignore_lable的原因
